share/templates/CMakeLists.txt

```diff
@@ -77,7 +77,10 @@ install(FILES
 
 install(FILES
       07-Band_and_Percussion/01-Concert_Band.mscz
-      07-Band_and_Percussion/02-Marching_Percussion.mscz
+      07-Band_and_Percussion/02-Marching_Band.mscz
+      07-Band_and_Percussion/03-Marching_Percussion_(battery).mscz
```

Comment:
Make it  `07-Band_and_Percussion/03-Marching_Percussion_\(battery\).mscz`, i.e. 'escape' the parens with a backslash